index.js

```diff
@@ -80,6 +80,28 @@ const parseInput = async input => {
 	return input;
 };
 
+const killChecker = async input => {
+	let i = 0;
+
+	return new Promise(resolve => {
```

Comment:
Almost anytime you want to use `new Promise`, there's a better way. You can just recursively await instead.

Reply:
In this case new promise is being used to convert the callback nature of setInterval (would also apply in the setTimeout format) into a promise that we can then await. Could you expand a bit on how we could go about this conversion without using new promise?

Reply:
Per https://github.com/sindresorhus/fkill/pull/37/files#r383127369, you use `setTimeout` instead and make it recursive by calling itself.

https://stackoverflow.com/questions/46242600/recursive-async-function-in-javascript